cluster/gce/gci/configure-helper.sh

```diff
@@ -990,6 +995,7 @@ function start-kube-apiserver {
   sed -i -e "s@{{admission_controller_config_volume}}@${admission_controller_config_volume}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_mount}}@${image_policy_webhook_config_mount}@g" "${src_file}"
   sed -i -e "s@{{image_policy_webhook_config_volume}}@${image_policy_webhook_config_volume}@g" "${src_file}"
+  sed -i -e "s@{{pillar\['enable_cache_mutation_detector'\]}}@${ENABLE_CACHE_MUTATION_DETECTOR}@g" "${src_file}"
```

Comment:
@cjcullen that sounds like it might be it. Where is the right place to define it?